x/protorev/keeper/rebalance.go

```diff
@@ -220,37 +222,43 @@ func (k Keeper) ExecuteTrade(ctx sdk.Context, route poolmanagertypes.SwapAmountI
 		return err
 	}
 
+	// Create and emit the backrun event and add it to the context
+	err = EmitBackrunEvent(ctx, pool, inputCoin, profit, tokenOutAmount, remainingTxPoolPoints, remainingBlockPoolPoints)
+	if err != nil {
+		return err
+	}
```

Comment:
Note that adding error checks can be state-breaking. However, it does not seem that `EmitBackrunEvent` actually ever returns an error so we can probably remove it

Reply:
Noted, resolved in https://github.com/osmosis-labs/osmosis/pull/4878/commits/f632b421b63d63f15008b49d60a9f961094d8af7 by removing the unused error return